oak_runtime/src/lib.rs

```diff
@@ -987,6 +987,11 @@ impl Runtime {
     ) -> Result<Option<NodeReadStatus>, OakStatus> {
         let half = self.abi_to_read_half(node_id, handle)?;
         self.validate_can_read_from_channel(node_id, &half)?;
+        self.introspection_event(EventDetails::MessageDequeued(MessageDequeued {
+            node_id: node_id.0,
+            channel_id: half.get_channel_id(),
+            acquired_handles: vec![],
+        }));
```

Comment:
Should this be here?  It seems like the event below covers the point when the message is actually dequeued.

Reply:
Shouldn't be, removed it